components/switch/switch-mixin.js

```diff
@@ -47,7 +47,7 @@ export const SwitchMixin = superclass => class extends FocusMixin(RtlMixin(super
 			}
 
 			.d2l-switch-container {
-				background-color: var(--d2l-switch-container-background-color, '#fff');
+				background-color: var(--d2l-switch-container-background-color, "#fff");
```

Comment:
I think this could just be `background-color: var(--d2l-switch-container-background-color, #ffffff);`, ya?